en/docs/code/extended_topics/data_loader/test_mini_dataloader.py

```diff
@@ -69,35 +73,38 @@ def test_job() -> tp.Numpy:
             shuffle_after_epoch=True,
         )
 
-        x, y = MiniDecoder(
-             miniRecord, name="d1"
-         )
+        x, y = MiniDecoder(miniRecord, name="d1")
 
-        initializer1 = flow.random_uniform_initializer(-1/28.0, 1/28.0)
+        initializer1 = flow.random_uniform_initializer(-1 / 28.0, 1 / 28.0)
         hidden = flow.layers.dense(
-                    x,
-                    500,
-                    activation=flow.nn.relu,
-                    kernel_initializer=initializer1,
-                    bias_initializer=initializer1,
-                    name="dense1",
-                )
-        initializer2 = flow.random_uniform_initializer(-np.sqrt(1/500.0), np.sqrt(1/500.0))
+            x,
+            500,
+            activation=flow.nn.relu,
+            kernel_initializer=initializer1,
+            bias_initializer=initializer1,
+            name="dense1",
+        )
+        initializer2 = flow.random_uniform_initializer(
+            -np.sqrt(1 / 500.0), np.sqrt(1 / 500.0)
+        )
         logits = flow.layers.dense(
-        hidden, 1, kernel_initializer=initializer2, bias_initializer=initializer2, name="dense2"
+            hidden,
+            1,
+            kernel_initializer=initializer2,
+            bias_initializer=initializer2,
+            name="dense2",
         )
-        loss = (y - logits)*(y-logits)
+        loss = (y - logits) * (y - logits)
 
         lr_scheduler = flow.optimizer.PiecewiseConstantScheduler([], [0.001])
         flow.optimizer.Adam(lr_scheduler).minimize(loss)
         return loss
 
 
 if __name__ == "__main__":
-    ckpt = flow.train.CheckPoint()
-    ckpt.init()
     loss = test_job()
     for i in range(0, 1000):
         loss = test_job()
-        if i % 100 == 0 : print("{}/{}:{}".format(i, 1000, loss.mean()))
+        if i % 100 == 0:
+            print("{}/{}:{}".format(i, 1000, loss.mean()))
     ckpt.save("./test_model")
```

Comment:
```suggestion
    oneflow.checkpoint.save("./test_model")
```
这里是不对的，应该用新接口，麻烦把cn里的代码同时也改一下。并跑一下这份代码测试下。